apps/prairielearn/src/pages/partials/plr/liveScoreboard.ejs

```diff
@@ -1,3 +1,23 @@
+<style>
```

Comment:
Add these to the CSS file. The words of Scott from 360 echo in your head: no embedded styling!